oe_corporate_blocks.post_update.php

```diff
@@ -180,3 +182,11 @@ function oe_corporate_blocks_post_update_30004(): void {
     $general_link->save();
   }
 }
+
+/**
+ * Import updated EU footer data, along with their translations.
```

Comment:
Only updates the data, the translations have to be imported manually.
```suggestion
 * Updates EU footer data.
```